packages/dds/map/src/interfaces.ts

```diff
@@ -13,7 +13,9 @@ import {
 
 /**
  * Type of "valueChanged" event parameter.
- * @alpha
+ * @privateRemarks
```

Comment:
These are all base interfaces for SharedMap and related logic, I don't think the private remarks here are necessary. These are the things we _should_ be exporting, where we really shouldn't be exporting the SharedMap class.

Reply:
they won't be checked in. this pr is purely to illustrate what would currently go public if we did do it.